src/pizzaOrder/Order.java

```diff
@@ -0,0 +1,60 @@
+package pizzaOrder;
+
+import java.util.List;
+
+public class Order {
+
+    private static int orderId = 10000;
+    private static int customerId = 0;
+    private List<Pizza> items;
+
+    public Order() {
+        orderId++;
+        customerId++;
+    }
+
+    public Order(List<Pizza> items) {
+        orderId++;
+        customerId++;
+        checkPizzaName(items);
+        this.items = items;
+
+    }
+
+    private void checkPizzaName(List<Pizza> items) {
+        for (int i = 0; i < items.size(); i++) {
+            if (items.get(i).getName().length() < 4 || items.get(i).getName().length() > 20) {
+                items.get(i).setName("customer_" + customerId + "_" + (i + 1));
+            }
+        }
+    }
+
+    public void displayingPizzaAttributes(Pizza pizza) {
```

Comment:
Please rename this method to `displayPizzaAttributes`

Reply:
Done